tests/systemd_testsuite/runner.pm

```diff
@@ -0,0 +1,73 @@
+# SUSE's openQA tests
+#
+# Copyright 2019-2020 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Run systemd upstream test cases
+# Maintainer: Martin Loviska <mloviska@suse.com>
+
+use Mojo::Base qw(systemd_testsuite_test);
+use testapi;
+
+my $test_hash;
+
+sub build_cmd {
+    my ($target, $args) = @_;
+    my @opts = ();
+    my $test = $args->{test};
+    my $dir = $args->{dir};
+
+    foreach my $k (keys %{$args->{make_opts}}) {
+        if ($args->{make_opts}->{$k}) {
+            push @opts, "$k=$args->{make_opts}->{$k}";
+        }
+    }
+
+    return sprintf('%s make -C %s%s %s', join(" ", @opts), ${dir}, ${test}, $target);
+}
+
+sub run {
+    my ($self, $args) = @_;
+    my $timeout = 900;
+    my $marker = " systemd test runner: >>> $args->{test} has finished <<<";
+    my $logs = qr[\/var\/tmp\/systemd-test.(\w+)\/];
+
+    $self->select_serial_terminal();
+
+    assert_script_run(build_cmd('clean', $args));
+    enter_cmd(build_cmd('setup', $args));
+
+    my $out = wait_serial(qr/make: Leaving directory.*/, 240, 0, record_output => 1);
```

Comment:
You should use `script_output()` instead of `enter_cmd()+wait_serial()`.

Reply:
Replace for both make targets (setup, run)